srcpkgs/mold/template

```diff
@@ -0,0 +1,45 @@
+# Template file for 'mold'
+pkgname=mold
+version=0.9.1
+revision=1
+build_style=gnu-makefile
+hostmakedepends="clang"
+makedepends="mimalloc-devel openssl-devel xxHash-devel tbb-devel zlib-devel"
+short_desc="High performance drop-in replacement for existing Unix linkers"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
+license="AGPL-3.0-or-later"
+homepage="https://github.com/rui314/mold"
+changelog="https://github.com/rui314/mold/releases"
+distfiles="https://github.com/rui314/mold/archive/refs/tags/v${version}.tar.gz"
+checksum=02b156de6cd2d94fea4eed9748a7c96955673d810ec672359f603f2f90e2990d
+_commit_hash=9a09c777d9460ebe7eb498d1cc0327915f8bbcdc
+
+# use system-wide installed mimalloc instead of the vendored one
+export SYSTEM_MIMALLOC=1
+
+pre_build() {
+	# gcc 10.2 doesn't fully support C++20
+ 	export CXX=clang++
+
+	# fix cross
+	if [ "$CROSS_BUILD" ]; then
+		CXXFLAGS+=" -target ${XBPS_CROSS_TRIPLET}"
+		CXXFLAGS+=" -I/usr/${XBPS_CROSS_TRIPLET}/usr/include/c++/10.2/${XBPS_CROSS_TRIPLET}"
```

Comment:
You're hardcoding the GCC version

Reply:
any way to avoid this? doing some `ls` with `sed` stuff maybe?

Reply:
Apparently GCC now works